README.md

```diff
@@ -252,11 +252,11 @@ Note: Although RetDec now supports a system-wide installation ([#94](https://git
     # FreeBSD (and other BSDs) do need cmake, python3, git, autotools. Perl is pre-installed in the OS but check its version.
     # Later versions may be available for each of the packages.
     # See what is installed:
-    sudo pkg info cmake python37 autotools
+    sudo pkg info cmake python autotools
     # Install/upgrade them:
     sudo pkg install cmake python37 autotools
+    cmake .. -DCMAKE_INSTALL_PREFIX=/usr/local (/usr/local is the standard installation path for software that is not part of a base [FreeBSD] operating system)
     ```
-  * `cmake .. -DCMAKE_INSTALL_PREFIX=<path>`
```

Comment:
Maybe we should just mention the following information in a comment? To let users who want to install RetDec into the system know that it has to be installed into `/usr/local`. What do you think? 
> Also making it clear that Retdec MUST be in "/usr/local" if not the command like "freebsd-update -r 12.1-RELEASE upgrade" will remove Retdec files. (as the Retdec installation files should not be in operating system directories)

Reply:
As FreeBSD is an operating system (not a kernel like Linux + GNU utilities) every package that is installed on top of the operating system goes into /usr/local. 

FreeBSD (fairly recently) has a great way of upgrading.. i.e. I was on FreeBSD 11.3 and upgraded to the current release 12.1-p4 (patch 4), all I have to do is (from the shell, preferably login as root not sudo);
# freebsd-update -r 12.1-RELEASE upgrade
It will examine my operating system (ignoring /usr/local) and work out what needs updating including merging configuration files, passwd, groups, startup scripts etc
If it sees anything in the actual operating system it does not recognise it prompts for it to be removed. One prompt for every file instance! i.e. So every file that was installed gets a prompt. It gets a bit tedious to keep typing `y` or `n`.
Also it is part of FreeBSD "standards" for packages/ports that ALL non-OS related software goes into /usr/local.
Software like llvm/clang has been integrated as part of the currently supported releases with good reason (dumping poor gcc).

Another lifetime ago LOL (back in 1999/2000) I saw that Apple were going to ditch their OS and take FreeBSD 9.0 + a Mach kernel. They recognised how so many enterprise companies were using it. BUT not many were really contributing much.

Apple pushed for FreeBSD to remove GNU gcc as it had the GPLvX license, you know llvm/clangs origins.

Netflix and Apple have contributed a huge amount to FreeBSD! As you probably know ALL of Netflix's infrastructure is based on FreeBSD... 

The reason I mention all of this is because FreeBSD is very structured in its approach to ensure rock solid performance. 

When I get around to it I should create a package for retdec and put it into the package database of FreeBSD. When that happens as part of the FreeBSD standard it MUST be installed in /usr/local. No ifs no buts!

I hope that makes sense...

I can just tone down the comment and say it is highly recommend to use /usr/local

Reply:
Thank you for the explanation. What I would suggest is to have the following instructions in the FreeBSD part:
```
* `cmake .. -DCMAKE_INSTALL_PREFIX=<path>` (use `/usr/local` if you wish to install RetDec system-wide)
```
I.e. the user can choose a different path (e.g. into their home directory), but when they want to install RetDec system-wide, they should choose `/usr/local`.

Reply:
Sounds goo to me..

Also take a look at this .. on github ...  NationalSecurityAgency/ghidra

Released last year!